pkgs/by-name/ra/rassumfrassum/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+let
+  pname = "rassumfrassum";
+  version = "0.2.1";
+in
+python3Packages.buildPythonApplication {
+  inherit pname version;
```

```suggestion
  pname = "rassumfrassum";
  inherit version;
```
since pname is only used once, it can probably just be inlined here.